### PR TITLE
Fix TypeScript types

### LINKED ISSUE #761: NotificationReceivedEvent complete method parameter has incorrect types
**Description:**

According to the documentation found [here](https://documentation.onesignal.com/docs/cordova-30-api-reference#setnotificationopenedhandler-function) and [here](https://documentation.onesignal.com/docs/sdk-notification-event-handlers#osnotificationreceivedevent-methods) the `NotificationReceivedEvent` `complete` method parameter should accept `OSNotification` or `null` but the [types](https://github.com/OneSignal/OneSignal-Cordova-SDK/blob/main/types/Notification.d.ts#L44) do not reflect that:

```TS
export interface NotificationReceivedEvent {
    complete        : (notification?: OSNotification) => void;
    getNotification : () => OSNotification;
}
```
`notification?: OSNotification` should be replaced by `notification: OSNotification | null`

**Steps to Reproduce Issue:**

Screenshot 2021-12-21 at 16 30 27

### PR DESCRIPTION
**Pull request by nunoarruda**
Fixes #761 and #726

---
This change is [Reviewable](https://reviewable.io/reviews/onesignal/onesignal-cordova-sdk/762)

### CODE REVIEW

**Comment by nunoarruda** [comment]
@jkasten2 @rgomezp @nan-li @Jeasmine can you folks please take a look at this when you have a chance? It's blocking us (and probably many more teams) from upgrading to v3.x.

Thanks in advance! 🙏

File: types/Notification.d.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export interface OSNotification {
 
 /* N O T I F I C A T I O N   &   I A M   E V E N T S */
 export interface NotificationReceivedEvent {
-    complete        : (notification?: OSNotification) => void;
+    complete        : (notification: OSNotification | null) => void;
```
Comment (jkasten2) [inline types/Notification.d.ts:44]: Since we allowed `undefined` we should continue supporting it, so we need to keep the `?` to allow that.

**Review by jkasten2 — CHANGES_REQUESTED** [review]
@nunoarruda Thanks for the PR! I have reviewed your PR and I have one comment on `undefined` that need to be address before we can merge this in.

File: types/Notification.d.ts (hunk 1 above)
Comment (nunoarruda) [reply types/Notification.d.ts]: @jkasten2 ok that makes sense. I've just added the `?`.

**Review by nan-li — APPROVED** [review]
LGTM

**Pull request merged**